### PR TITLE
Master co edi bfr

### PR DESCRIPTION
**Pull request by bfr-o**
Adapt co edi to account_edi

See upgrade script here : odoo/upgrade#1946
PR enterprise : https://github.com/odoo/upgrade/pull/1946

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account_edi_facturx/models/account_edi_format.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def _post_invoice_edi(self, invoices, test_mode=False):
         res = {}
         for invoice in invoices:
             attachment = self._export_facturx(invoice)
-            res[invoice] = {'attachment': attachment}
+            res[invoice] = {'success': True, 'attachment': attachment}
```
Comment (jco-odoo) [inline addons/account_edi_facturx/models/account_edi_format.py:33]: @bfr-o Are you sure you want to have to put success True here even if the format does not need webservices?

Comment (bfr-o) [reply addons/account_edi_facturx/models/account_edi_format.py:33]: Yes, success is not related to the web-service.

NB: the flow of 'post' is slightly modified in this PR (separate the successful posting from the return of an attachment, rather use a special key, like for the cancel flow, could be useful). I give more details in the commit message.

File: addons/l10n_nl_edi/models/account_edi_format.py [hunk 2]
```diff
@@ -127,7 +127,7 @@ def _post_invoice_edi(self, invoices, test_mode=False):
 
         invoice = invoices  # no batch ensure that there is only one invoice
         attachment = self._export_nlcius(invoice)
-        return {invoice: {'attachment': attachment}}
+        return {invoice: {'succes': True, 'attachment': attachment}}
```
Comment (jco-odoo) [inline addons/l10n_nl_edi/models/account_edi_format.py:130]: careful here

**Comment by jco-odoo** [comment]
@robodoo r+ rebase-ff

**Comment by smetl** [comment]
@robodoo r+

**Pull request closed without merging**